scaper/util.py

```diff
@@ -143,7 +145,118 @@ def _populate_label_list(folder_path, label_list):
     label_list.sort()
 
 
-def _trunc_norm(mu, sigma, trunc_min, trunc_max):
+def _check_random_state(seed):
+    """Turn seed into a np.random.RandomState instance
+
+    Parameters
+    ----------
+    seed : None | int | instance of RandomState
+        If seed is None, return the RandomState singleton used by np.random.
+        If seed is an int, return a new RandomState instance seeded with seed.
+        If seed is already a RandomState instance, return it.
+        Otherwise raise ValueError.
+    """
+    if seed is None or seed is np.random:
+        return np.random.mtrand._rand
+    if isinstance(seed, (numbers.Integral, np.integer)):
+        return np.random.RandomState(seed)
+    if isinstance(seed, np.random.RandomState):
+        return seed
+    raise ValueError('%r cannot be used to seed a numpy.random.RandomState'
+                     ' instance' % seed)
+
+
+def _const(item, random_state):
```

Comment:
I feel like the names for the sampling functions are a little too short/ambiguous. Can we add `_sample` as a prefix to all of these? This function would be renamed `_sample_const`.